### DISCUSSION TITLE
Web Languages

### DISCUSSION THREAD

**Discussion by WebShells** [opensourcepos/opensourcepos#3664] [Team Posts]
Hello Guys,
Can i know what are the preferred web languages for the website to be translated too ? I wish we can add them all, but that's a lot of huge work since i'll be translating each string 1 by 1... So if you can mention top 10 languages to be used in the web, it will be great! 
Will be a drop down icon next to main menu in order for user to choose his language, and on the other hand each user will be redirected to preferred language depending on browser setting...

**Comment by daN4cat** [comment]
According to some statistics related to number of people and language they speak: Chinese, Spanish, English, Hindi, Arabic.... https://en.wikipedia.org/wiki/List_of_languages_by_number_of_native_speakers

**Comment by WebShells** [comment]
Hmmm sounds less work, great news :p! ill translate menu and main things for now...

**Comment by objecttothis** [comment]
Perhaps it would be beneficial and less work for you to make a new weblate project for translation.  The reason I'm thinking is that there isn't just the initial translation, but also the maintenance of the translations (corrections, new phrases, etc) that needs to be done.

As far as I can tell, the top 10 languages are going to be English, French, German, Spanish, Russian, Arabic (Levant?), Chinese, Portuguese, Indonesian and Japanese.

I also just found this website that gives stats on the most used languages on the internet: https://speakt.com/top-10-languages-used-internet/
